stream/cards.md

```diff
@@ -2,7 +2,7 @@
 
 ## Card Linking
 
-To be able to participate in your program, users must link their cards to it first. You can implement this in your application using Fidel API's SDKs and APIs.
+To be able to participate in your program, users must link their cards to it first. You can implement this in your application using Fidel API's [SDKs](/stream/sdks) and APIs.
```

Comment:
Will the link work if it’s written like this? Should it not be a full link, like I see for the other links in the pages? (ex: `https://fidelapi.com/docs/stream/sdks`)

Reply:
Relative links should work, but you are right, the relative links I added were incorrect.
To fix it, I would need to know the exact path and file name of the page that describes the SDKs.
Are you planning on adding a page where we have some text regarding what is in the "SDKs" chapter's subpages? What will be the full path of that file?

Reply:
Relative links should work, but the relative links I added were incorrect, thanks for bringing it to my attention, fixed now.